pkg/util/cgroups/utils.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package cgroups
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"time"
+)
+
+// UserHZToNano holds the divisor to convert HZ to Nanoseconds
+// It's safe to consider it being 100Hz in userspace mode
+const (
+	UserHZToNano uint64 = uint64(time.Second) / 100
+)
+
+func uint64Ptr(v uint64) *uint64 {
+	return &v
+}
+
+func float64Ptr(v float64) *float64 {
+	return &v
+}
```

Comment:
nit - I am surprised we don't have this in a common package

Reply:
We probably do but this package is meant to be self-contained if we want to move it to a dedicated repo.